assets/js/components/notifications/AdBlockingRecoverySetupSuccessBannerNotification.js

```diff
@@ -85,35 +90,41 @@ export default function AdBlockingRecoverySetupSuccessBannerNotification() {
 	}
 
 	return (
-		<BannerNotification
-			id={ AD_BLOCKING_RECOVERY_SETUP_SUCCESS_NOTIFICATION_ID }
-			className="googlesitekit-ad-blocking-recovery-notification"
-			title={ __(
-				'You successfully enabled the ad blocking recovery message',
-				'google-site-kit'
-			) }
-			description={ createInterpolateElement(
-				__(
-					'Make sure to also create the message in <a>AdSense</a>, otherwise this feature won’t work.',
+		<Fragment>
+			<SurveyViewTrigger
+				triggerID="abr_setup_completed"
+				ttl={ DAY_IN_SECONDS }
+			/>
+			<BannerNotification
```

Comment:
This would trigger the survey even if the banner notification were dismissed. How about we trigger the survey from the `handleView` function here instead since we're already tracking an event there. The component helper is less useful in this context since the banner component has additional logic internally.

Reply:
Oh, fair enough! I went with that first then realised I could use the component, but that probably works fine here 👍🏻